src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -199,12 +206,15 @@ protected function getContainerBuilder()
         if (!$kernel->isDebug() || !(new ConfigCache($kernel->getContainer()->getParameter('debug.container.dump'), true))->isFresh()) {
             $buildContainer = \Closure::bind(function () { return $this->buildContainer(); }, $kernel, get_class($kernel));
             $container = $buildContainer();
-            $container->getCompilerPassConfig()->setRemovingPasses(array());
-            $container->compile();
         } else {
             (new XmlFileLoader($container = new ContainerBuilder(), new FileLocator()))->load($kernel->getContainer()->getParameter('debug.container.dump'));
+            $container->setParameter('container.build_hash', $hash = ContainerBuilder::hash(__METHOD__));
+            $container->setParameter('container.build_id', hash('crc32', $hash.time()));
```

Comment:
why these changes ?

Reply:
```
The service "cache.adapter.apcu" has a dependency on a non-existent parameter "container.build_id".

The service "security.authentication.listener.anonymous.main" has a dependency on a non-existent parameter "container.build_hash".
```

should they be dumped instead? cc @nicolas-grekas